soccer/VisionReceiver.cpp

```diff
@@ -36,15 +36,21 @@ void VisionReceiver::run() {
 
     // Create vision socket
     if (simulation) {
+        if (!socket.bind(SimVisionPort, QUdpSocket::ShareAddress)) {
+            throw runtime_error("Can't bind to shared vision port");
+        }
+        multicast_add(&socket, SharedVisionAddress);
         // The simulator doesn't multicast its vision.  Instead, it sends to two
         // different ports.
         // Try to bind to the first one and, if that fails, use the second one.
+        /*
         if (!socket.bind(SimVisionPort)) {
```

Comment:
Maybe this can be completely removed, we aren't planning on using the old simulator again...